src/common.js

```diff
@@ -96,6 +96,44 @@ var ShellString = function (stdout, stderr, code) {
   } else {
     that = new String(stdout);
     that.stdout = stdout;
+
+    that.forEach = function (callback, opts) {
```

Comment:
Will this override the `.forEach()` that's part of the Array type by default (I'm thinking of `ls()` here)? If so, how would this compare in performance?